### PR TITLE
Pigeon readme update supported chains

### PR DESCRIPTION
**Pull request by taariq**
# Related Github tickets
This is an update to support Binance Smart Chain in the pigeon README.

# Background
This is an update to support Binance Smart Chain in the pigeon readme so that new installs will support Binance Smart Chain relay service.

# Testing completed
- [ ] pigeons have already verified updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,9 +123,18 @@ evm:
   eth-main:
     chain-id: 1
     base-rpc-url: ${ETH_RPC_URL}
-    keyring-pass-env-name: ETH_PASSWORD
+    keyring-pass-env-name: "ETH_PASSWORD"
```
Comment (verabehr) [inline README.md:126]: why are we changing this? I have it unquoted in my config file

File: README.md [hunk 2]
```diff
@@ -109,9 +123,18 @@ evm:
...
     signing-key: ${ETH_SIGNING_KEY}
     keyring-dir: ~/.pigeon/keys/evm/eth-main
+    gas-adjustment: 1.2
+
+  bsc-main:
+    chain-id: 56
+    base-rpc-url: ${BSC_RPC_URL}
+    keyring-pass-env-name: "BSC_PASSWORD"
```
Comment (verabehr) [inline README.md:134]: see above, I don't think we need "" here

File: README.md (hunk 1 above)
Comment (taariq) [reply README.md]: reverted.

File: README.md (hunk 2 above)
Comment (taariq) [reply README.md]: reverted

**Review by verabehr — APPROVED** [review]

**Pull request merged**